azure-pipelines.yml

```diff
@@ -151,7 +129,7 @@ jobs:
       conda env update --file $CONDA_ENV
       pip uninstall -yq mne
       pip install -ve .
-      pip install pytest pytest-cov pytest-timeout pytest-sugar pytest-xdist flake8 codecov
+      pip install -r requirements_testing.txt
```

Comment:
lost `codecov` on this line...  why not include it in `requirements_testing.txt`?  Looks like most other lines using `requirements_testing.txt` also need `codecov`

Reply:
You only need `codecov` if you are pushing coverage, i.e., you are a CI provider. devs don't need it